src/groupByKey.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.groupByKey = groupByKey;
+function groupByKey(items, key) {
+    var nObj = {};
```

Comment:
Consider using `const` instead of `var` for declaring `nObj` since it is not reassigned later. This helps to communicate intent and ensures the variable is not accidentally modified elsewhere.